src/dispatch.rs

```diff
@@ -100,17 +104,26 @@ impl Dispatcher {
         if self.connections.contains_key(&req.db_name[..]) {
             return Ok("".into());
         }
-        match IdbStore::new(&req.db_name[..]).await {
-            Err(e) => {
-                return Err(format!("Failed to open \"{}\": {}", req.db_name, e));
-            }
-            Ok(v) => {
-                if let Some(kv) = v {
-                    self.connections
-                        .insert(req.db_name.clone(), Box::new(dag::Store::new(Box::new(kv))));
-                }
+        let idb_store = match IdbStore::new(&req.db_name[..]).await {
+            Err(e) => return Err(format!("Failed to open \"{}\": {}", req.db_name, e)),
+            Ok(None) => return Err(format!("Failed to open \"{}\"", req.db_name)),
+            Ok(Some(idb_store)) => {
+                let _client_id = match init_client_id(&idb_store).await {
```

Comment:
I think this should be like:

if let Err(e) = init_client_id(&idb_store).await {
    return Err(...);
}

Reply:
(Since you don't care about `_client_id`.)

Reply:
We will care about client_id in the future